packages/frontend/src/components/RefreshServer/JobDetailsDrawer/index.tsx

```diff
@@ -42,54 +73,50 @@ const JobDetailsDrawer: FC = () => {
             title={
                 <RefreshStepsHeadingWrapper className={Classes.DIALOG_HEADER}>
                     <Icon
-                        icon={refreshStatusInfo(activeJob?.jobStatus).icon}
+                        icon={jobStatusLabel(activeJob?.jobStatus).icon}
                         size={18}
                     />
                     <div>
                         <RefreshStepsTitle>
-                            {refreshStatusInfo(activeJob?.jobStatus).title}
+                            {jobStatusLabel(activeJob?.jobStatus).label}
                         </RefreshStepsTitle>
                         {hasSteps && (
                             <StepsCompletionOverview>{`${
                                 runningStepsInfo(activeJob?.steps)
                                     .numberOfCompletedSteps
-                            } steps complete `}</StepsCompletionOverview>
+                            }/${
+                                runningStepsInfo(activeJob?.steps).totalSteps
+                            } steps complete - ${durationSince(
```

Comment:
The runningStepsInfo is already returning this info for us.

https://github.com/lightdash/lightdash/blob/aa513d90fe54ecf73b40bc8190d91794369521ee/packages/frontend/src/hooks/useRefreshServer.tsx#L63

we can either just use `${ runningStepsInfo(activeJob?.steps).completedStepsMessage}` or lets get rid of that part in the function :)

Reply:
Good catch!